tests/test_project/tests/test_test_utils.py

```diff
@@ -40,16 +42,18 @@ def test_deep_merge_dicts(self):
         self.assertDictEqual(deep_merge_dicts(dict1, dict2), merged)
 
     def test_time_logging_runner(self):
+        setattr(settings, 'OPENWISP_SLOW_TEST_THRESHOLD', [0.0, 0.0])
```

Comment:
This is fragile: if an assertion fails, the last line won't be executed and it will change the behavior of this feature for all the tests, you should always use `mock.patch` or `override_settings` (from django) in these cases.

Reply:
`override_settings` works great. Thanks! :smile: